docs/auth.md

```diff
@@ -0,0 +1,149 @@
+# Introduction
+
+kcp implements the same RBAC-based authorization scheme as Kubernetes.
+Other authorization schemes (i.e. ABAC) are currently not supported.
+Generally, the same role and role binding priniciples apply the same way as in Kubernetes.
+
+However RBAC rules are always declared within the context of a concrete workspace.
+Rules created in a workspace apply to requests against this workspace independently from the workspace type.
+This is what resemembles Kubernetes authorization model.
+In addition, rules can play a role in authorization to requests to other workspaces. The details will be described below.
+This is true for all non-system workspace types, including root, organizational and enduser.
+The exceptions are system namespaces. These can only be accessed by shard-local admins.
```

Comment:
These sentences sound confusing. IMO it's much easier to explain how a request is authorized rather than explaining how and where rules are used. Here both is even mixed in one paragraph.